lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -96,9 +108,23 @@ def rabbitmq_settings
 
         s[:timeout] = @connection_timeout || 0
         s[:heartbeat] = @heartbeat || 0
-        s[:tls] = @ssl if @ssl
-        s[:tls_certificate_path] = @tls_certificate_path || ""
-        s[:tls_certificate_password] = @tls_certificate_password || ""
+
+        # This MUST be passed as a string or the cert options will be ignored
```

Comment:
I think you can remove this comment now, right? or just make it set on the `s[:tls]` line instead.